integration_test.go

```diff
@@ -28,11 +28,11 @@ func consecAll(n string, char rune) bool {
 	return true
 }
 
-func consec(n, sn string) bool {
+func consecHelper(n, sn string) bool {
```

Comment:
We need to determine if two hash key ranges are consecutive because I try to merge shards but the keys are too big to fit in numbers.